Cargo.toml

```diff
@@ -22,7 +22,7 @@ exclude = [
 too_many_arguments = "allow"
 
 # This crate is particularly sensitive to compiler optimizations
-[profile.dev.package.turbo-tasks-memory]
+[profile.dev.package.turbo-persistence]
 opt-level = 1
```

Comment:
I did run unit tests with `opt-level = 3`, but it seems like we get the vast majority of the benefit from just moving from `opt-level = 0` to `1`. This option is only used when the package is built in `dev`.

Aside from compilation time concerns, usually you don't want to do any more optimizations than needed in dev because higher optimization levels can mess with debuggers.